lib/pages/create_blog_ui

```diff
@@ -0,0 +1,194 @@
+                                      if (_image == null) {
+                                        ScaffoldMessenger.of(context)
+                                            .showSnackBar(SnackBar(
+                                          content: Text(
+                                            "Choose a Picture to Post",
+                                            style:
+                                                TextStyle(color: Colors.white),
+                                          ),
+                                          backgroundColor: Colors.red,
+                                        ));
+                                        return;
+                                      }
+                                      if (formKey.currentState.validate()) {
+                                        print(ReviewTextController.text);
+                                        User user = await FirebaseAuth
+                                            .instance.currentUser;
+                                        BlogDetails blogDetails =
+                                            new BlogDetails(
+                                                user.uid,
+                                                ReviewTextController.text,
+                                                "test");
+                                        CrudMethods.saveBlogData(
+                                            File(_image.path), blogDetails);
+                                        Navigator.push(
+                                          context,
+                                          MaterialPageRoute(
+                                              builder: (context) =>
+                                                  BlogsPage()),
+                                        );
+                                      }
+                                    },
+                                    text: "Post",
+                                  )
+                                ]),
+                          ),
+                        )),
+                  )));
+        });
+  }
+
+  void _showPicker(context) {
+    showModalBottomSheet(
+        context: context,
+        builder: (BuildContext bc) {
+          return SafeArea(
+            child: Container(
+              child: new Wrap(
+                children: <Widget>[
+                  new ListTile(
+                      leading: new Icon(Icons.photo_library),
+                      title: new Text('Gallery'),
+                      onTap: () {
+                        _imgFromGallery();
+                        Navigator.of(context).pop();
+                      }),
+                ],
+              ),
+            ),
+          );
+        });
+  }
+
+  _imgFromGallery() async {
+    XFile image =
+        await _picker.pickImage(source: ImageSource.gallery, imageQuality: 50);
+    setState(() {
+      _image = image;
+    });
+  }
+}
```

Comment:
No new line at the end of file. Just add a new line here

Reply:
Done Rajat.